packages/runtime-core/src/components/Teleport.ts

```diff
@@ -9,7 +9,7 @@ import {
   traverseStaticChildren
 } from '../renderer'
 import { VNode, VNodeArrayChildren, VNodeProps } from '../vnode'
-import { isString, ShapeFlags } from '@vue/shared'
+import { isString, isSVGTag, ShapeFlags } from '@vue/shared'
```

Comment:
FYI `isSVGTag` is for compiler only and should be avoided in runtime code since it bloats the bundle size.